stretch-renderer/src/main/java/com/esri/arcgisruntime/sample/stretchrenderer/ParametersDialogFragment.java

```diff
@@ -201,15 +205,20 @@ public void onStartTrackingTouch(SeekBar seekBar) { }
       @Override
       public void onStopTrackingTouch(SeekBar seekBar) { }
     });
+
     mPercentClipMaxSeekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
       @Override
       public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
         mPercentClipMax = progress;
         updateSeekBar(mPercentClipMaxSeekBar, mPercentClipMax, mCurrPercentClipMaxTextView);
-        // move min to match max if min goes above max
-        if (mPercentClipMin > mPercentClipMax) {
-          mPercentClipMin = mPercentClipMax;
-          updateSeekBar(mPercentClipMinSeekBar, mPercentClipMin, mCurrPercentClipMinTextView);
+        if (mPercentClipMin + mPercentClipMax > 100) {
+            // constrain min + max <= 100
+            mPercentClipMin = 100 - mPercentClipMax;
+            updateSeekBar(mPercentClipMinSeekBar, mPercentClipMin, mCurrPercentClipMinTextView);
+        } else if (mPercentClipMin > mPercentClipMax){
```

Comment:
min and max define the percentile cutoff below and above respectively. Max is not necessarily > min. we should not put max > min as a constrain.